03-task-manager/starter/app.js

```diff
@@ -4,18 +4,21 @@ const app = express();
 const {getAllTasks} = require('./controllers/tasks.js')
 const connectDB = require('./db/connect.js')
 require('dotenv').config()
+const notFound = require('./middleware/not-found.js');
+const asyncWrapper = require('./middleware/async.js')
+const errorHandlerMiddleware = require('./middleware/error-handler.js')
 
 //middleware 
+app.use(express.static('./public'))
 app.use(express.json())
 
-//routes
-app.get('/hello', (req,res) => {
-    res.send("Task Manager App")
-})
+app.use(notFound)
```

Comment:
This has to be below the task routes on 19. Right now if you try to use any of the tasks routes, it's always going to use the notFound middleware and return a 404. Express is matching the first route it's finding from top to bottom since the notFound is first and is a catch-all it's always going to return that 404